### PR TITLE
add no-prompt option for app install

### PR DESCRIPTION
**Pull request by gitlawr**
The option will be helpful for scripting using cli.
Address issue https://github.com/rancher/rancher/issues/16177

### CODE REVIEW

File: cmd/app.go [hunk 1]
```diff
@@ -145,6 +145,10 @@ func AppCommand() cli.Command {
 						Name:  "version",
 						Usage: "Version of the template to use",
 					},
+					cli.BoolFlag{
+						Name:  "no-prompt",
+						Usage: "Suppress asking questions and use the default values when required answers are not provided",
```
Comment (alena1108) [inline cmd/app.go]: what is the default value?

File: cmd/app.go [hunk 2]
```diff
@@ -561,8 +565,9 @@ func templateInstall(ctx *cli.Context) error {
 			return err
 		}
 
+		interactive := !ctx.Bool("no-prompt")
 		answers := make(map[string]string)
-		err = processAnswers(ctx, c, templateVersion, answers, true)
+		err = processAnswers(ctx, c, templateVersion, answers, interactive)
```
Comment (alena1108) [inline cmd/app.go]: if no-prompt is set to false by default, the old behavior of passing true to processAnswers by default won't be preserved.

File: cmd/app.go (hunk 1 above)
Comment (gitlawr) [reply cmd/app.go]: False for bool value

File: cmd/app.go (hunk 2 above)
Comment (gitlawr) [reply cmd/app.go]: There's a negative above, interactive == !no-prompt

Comment (alena1108) [reply cmd/app.go]: ah, my bad

File: cmd/app.go (hunk 1 above)
Comment (alena1108) [reply cmd/app.go]: shall we list the default value in help? there is a cli flag for it, called default value or something

**Pull request merged**